ovirtlago/__init__.py

```diff
@@ -210,15 +210,24 @@ def _create_rpm_repository(
         repo_names,
         custom_sources=None,
         projects_list=None,
+        dist_out=None,
     ):
 
         if not projects_list:
             projects_list = PROJECTS_LIST
 
+
         custom_sources = custom_sources or []
 
         rpm_dirs = []
+
+        dist_output = self.paths.internal_repo()
+        if (dist_out):
+            dist_output = dist_out
```

Comment:
Also, I'd put the assignment at the start of the function (or closer to where it's used)